ui/components/atoms/A012-ListItem/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+
+import ListItem from '@material-ui/core/ListItem'
+
+import schema from './schema'
+
+const A012 = props => {
+  const {button, children} = props
+
+  return (
+    <ListItem button={button}>
```

Comment:
for the purposes of the mobile navigation list we want to make I think we should hardcode this button as true ie not assign a value to it.

Reply:
updated ListItem